src/libs/ajax/DataRepo.ts

```diff
@@ -17,26 +17,11 @@ export type SnapshotBuilderDomainOption = {
   root: SnapshotBuilderConcept;
 };
 
-export interface ProgramDataOption {
+export interface SnapshotBuilderProgramDataOption {
   kind: 'range' | 'list';
-  id: number;
```

Comment:
Why remove id from options?

Reply:
The original plan was that all options would have an ID, which in the OMOP world corresponds to the concept ID for the property. As it turns out, the program data options are (currently) all values that are stored directly on Person. So there is no concept ID for them; the unique property for them is their name, which the backend can use to map to the database table's column name. So ID isn't needed here.